docs/components/accordion/accordion-bordered-example.code

```diff
@@ -1,8 +1,8 @@
-<jazz-accordion [showBorder]="true">
+<jazz-accordion [showBorder]="true" [multiSelect]="true">
```

Comment:
Default examples should be multi-select = false, when multiselect is not specified it should default to false as well

Reply:
The current behavior in master branch for Design-System project has single-select as default, whereas Design-System-Angular has multi-select as default without markers for "multi-select".

As discussed, I will change the default behavior for multi-select accordions in Design-System-Angular to match that of Design-System so that when "multi-select" is not specified, it would default to "single-select".